spotifyapi/endpoints/__init__.py

```diff
@@ -4,10 +4,11 @@
 from .player import PlayerEndpoint
 from .track import TrackEndpoint
 from ..models.token import Token
+from .user import UserEndpoint
```

Comment:
Move this after the .track import. They should be alphabetical but all of the single dot imports should come before the double dot